sources/packages/web/src/components/students/StudentDocumentList.vue

```diff
@@ -1,70 +1,73 @@
 <template>
-  <full-page-container>
-    <div class="category-header-medium-small mb-4">My received documents</div>
-    <div v-if="ministryDocuments.length">
-      <div
-        class="document-box mb-4"
-        v-for="ministryDocument in ministryDocuments"
-        :key="ministryDocument"
-      >
+  <v-card>
+    <v-container>
+      <h3 class="category-header-medium-small mb-4">My received documents</h3>
+      <div v-if="ministryDocuments.length">
         <div
-          class="file-label"
-          @click="fileUtils.downloadStudentDocument(ministryDocument)"
+          class="document-box mb-4"
+          v-for="ministryDocument in ministryDocuments"
+          :key="ministryDocument.uniqueFileName"
         >
-          <span class="mr-4">
-            <v-icon icon="fa:far fa-file-alt" size="20"></v-icon
-          ></span>
-          <span>{{ ministryDocument.fileName }}</span>
+          <div
+            class="file-label"
+            @click="fileUtils.downloadStudentDocument(ministryDocument)"
+          >
+            <span class="mr-4">
+              <v-icon icon="fa:far fa-file-alt" size="20"></v-icon
+            ></span>
+            <span>{{ ministryDocument.fileName }}</span>
+          </div>
         </div>
       </div>
-    </div>
-    <div v-else>
-      <p>
-        Your documents from StudentAid BC are shown here. At the moment there
-        are no documents to display.
-      </p>
-    </div>
-    <div class="category-header-medium-small mb-4">My uploaded documents</div>
-    <div v-if="studentDocuments.length">
-      <div
-        class="document-box mb-4"
-        v-for="studentDocument in studentDocuments"
-        :key="studentDocument"
-      >
+      <div v-else>
+        <p>
+          Your documents from StudentAid BC are shown here. At the moment there
+          are no documents to display.
+        </p>
+      </div>
+      <h3 class="category-header-medium-small mb-4">My uploaded documents</h3>
+      <div v-if="studentDocuments.length">
```

Comment:
Just as information, when the `div` are used only for the sake of adding the `v-if` we can switch the `div` for a `template`
Just a recommendation: https://vuejs.org/guide/essentials/conditional.html#v-if-on-template